openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -38,12 +38,20 @@
 # All optimizers that require an initial run
 run_required = ['NSGA2', 'ALPSO']
 
-CITATIONS = """
-@phdthesis{hwang_thesis_2015,
-  author       = {John T. Hwang},
-  title        = {A Modular Approach to Large-Scale Design Optimization of Aerospace Systems},
-  school       = {University of Michigan},
-  year         = 2015
+CITATIONS = """@article{Hwang_maud_2018
+ author = {Hwang, John T. and Martins, Joaquim R.R.A.},
+ title = {A Computational Architecture for Coupling Heterogeneous
```

Comment:
Learned recently that putting quotation marks around the curly braces in the title field will preserve capitalization of the title (as is done in the previous citation).